rootfs-builder/template/config_template.sh

```diff
@@ -6,3 +6,6 @@
 OS_VERSION=${OS_VERSION:-DEFAULT_VERSION}
 
 PACKAGES="systemd iptables udevlib.so"
+
+INIT_PROCESS=systemd
+ARCH_EXCLUDE_LIST=()
```

Comment:
Also, you could add a comment explaining this should be zero or more "uname -m" architectures the distro should not be built for.